dockerfile_parse/util.py

```diff
@@ -239,6 +239,10 @@ def substitute_vars(val):
         #   LABEL/ENV "name"="value" ["name"="value"...]
         # Each word is a key=value pair.
         for k_v in words:
+            if '=' not in k_v:
```

Comment:
I'm curios about cases:
* LABEL key=inside_key=value
* LABEL "key=value"
* LABEL something"key=value"

Reply:
and LABEL key==value

Reply:
All of those are valid LABELs according to docker:
```Dockerfile
FROM fedora:30

LABEL key=inside_key=value
LABEL "key2=value"
LABEL key"3=value"
LABEL key4==value
```

```bash
$ docker build -t testimage .
$ docker inspect testimage
```

```json
...

"Labels": {
    "key": "inside_key=value",
    "key2": "value",
    "key3": "value",
    "key4": "=value"
}
...
```

Reply:
Ok, thank you!